common/pom.xml

```diff
@@ -338,22 +333,10 @@
                         </manifestEntries>
                     </archive>
                 </configuration>
-            </plugin>
-
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-jar-plugin</artifactId>
-                <version>2.4</version>
-                <configuration>
-                    <archive>
-                        <manifestEntries>
-                            <Go-Version>${go.version}</Go-Version>
```

Comment:
I see that you've writing the version to the manifest entry. 
Who's doing this now? I don't see it in the root pom